services/m3dbnode/server/server.go

```diff
@@ -132,6 +132,9 @@ func Run(runOpts RunOptions) {
 			SetLimitCheckEvery(cfg.Filesystem.ThroughputCheckEvery)).
 		SetWriteNewSeriesAsync(cfg.WriteNewSeriesAsync).
 		SetWriteNewSeriesBackoffDuration(cfg.WriteNewSeriesBackoffDuration)
+	if lruConfig := cfg.Cache.SeriesConfiguration().LRU; lruConfig != nil {
```

Comment:
nit: We have usually been naming local vars with the word "config" in them to "cfg" so this would be `lruCfg` to be slightly more consistent.  Don't feel too strongly about it, but would be more consistent.

Reply:
done